python/fbprophet/make_holidays.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pandas as pd
+import fbprophet.hdays as hdays_part1
+import holidays as hdays_part2
+
+
+def make_holidays(year_list, country_list):
+    """Make dataframe of holidays for given years and countries
+
+    Parameters
+    ----------
+    year_list: a list of years
+    country_list: a list of countries
+
+    Returns
+    -------
+    Dataframe with 'ds' and 'holiday', which can directly feed
+    to 'holidays' params in Prophet
+    """
+
+    if isinstance(country_list, str):
+        country_list = [country_list]
+    all_hdays = []
+    for country in country_list:
```

Comment:
Question: Are there overlapping holiday names across countries? If so, do they have identical dates?
If there are overlapping holidays across countries, we want to be sure to grab only one copy of them if someone passes in a list of countries.